pkg/operation/cloudbotanist/gcpbotanist/infrastructure.go

```diff
@@ -123,8 +120,8 @@ func (b *GCPBotanist) generateTerraformBackupConfig() map[string]interface{} {
 			"project": b.Project,
 		},
 		"bucket": map[string]interface{}{
-			"name": fmt.Sprintf("%s-%s", b.Shoot.SeedNamespace, utils.ComputeSHA1Hex([]byte(b.Shoot.Info.Status.UID))[:5]),
+			"name": b.Operation.BackupInfrastructure.Name,
```

Comment:
What will happen to the existing buckets? Will they be recreated? How does Terraform behave?